packages/xforms-engine/src/client/validation.ts

```diff
@@ -204,4 +204,4 @@ export interface NullValidationState {
 export type NodeValidationState =
 	| AncestorNodeValidationState
 	| LeafNodeValidationState
-	| NullValidationState;
+	| NullValidationState; // eslint-disable-line @typescript-eslint/no-redundant-type-constituents
```

Comment:
@garethbowen I believe you added this type, but it's considered redundant. I left it because it still adds some clarity to the code. Does it make sense?